insert_products.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+    INSERT INTO products (name, description, price_cents) 
+    VALUES ('PayWall', 'Its a gacha', 265);
+
+    COMMIT;
+
+EXCEPTION
+    WHEN OTHERS THEN
+        ROLLBACK;
+        RAISE NOTICE 'Error at inserting new customer.';
```

Comment:
Pequeno erro de digitação:
```suggestion
        RAISE NOTICE 'Error at inserting new product.';
```